src/pages/CatShow.js

```diff
@@ -2,11 +2,17 @@ import React, { Component } from 'react';
 
 class CatShow extends Component {
   render() {
+
+    const { cat } = this.props
     return (
       <>
-        <h1>Check out this cat.</h1>
+          <h1>Check out this purrdy cat.</h1>
+            <p>Kitty name: {cat ? cat.name : null }</p>
+            <p>Kitty oldnes: {cat ? cat.age : null}</p>
+            <p>Kitty like: {cat ? cat.enjoys : null}</p>
+            { cat ? <img src={cat.image} alt="cats are sick" width=" 200px"/> : null}
```

Comment:
You may want to think about wrapping this all in a a single ternary and have the ternary display some html if there is no data...? Food for thought.